src/Symfony/Component/VarDumper/Caster/DbaCaster.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Caster;
+
+use Symfony\Component\VarDumper\Cloner\Stub;
+
+/**
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ */
+class DbaCaster
+{
+    public static function castDbaConnection(\Dba\Connection $dba, array $a, Stub $stub, bool $isNested): array
+    {
+        // not yet possible to gather information about the connection object
```

Comment:
did you try casting the object to an integer?
what does dba_list return now?
this should be reported to php-src, it's a regression

Reply:
dba_list() now returns an array with a starting index at 0. I prepared the issue and then it got out my head. I'll report it, thanks!